src/main/java/cms/gov/madie/measure/resources/MeasureController.java

```diff
@@ -163,6 +164,19 @@ public ResponseEntity<Measure> updateMeasure(
     return response;
   }
 
+  @DeleteMapping("/measures/{id}/delete")
+  public ResponseEntity<Measure> deactivateMeasure(
+      @PathVariable("id") String id, Principal principal) {
+    if (!StringUtils.hasLength(id)) {
```

Comment:
Swapping in Apache StringUtils, this will need to be replaced. Suggest `isBlank`.